client/src/components/Auction.jsx

```diff
@@ -40,7 +37,7 @@ class Auction extends Component {
       dispatch(Auctions.fetchingAnAuction(false));
       dispatch(Auctions.fetchAuctionErrored(true, err));
     });
-    //lala:
+
     if (user.username) {
```

Comment:
Agreed!! So which line of code do you think I could improve? I'm a bit confused.....